docs/schema/2.0/manifest.schema.json

```diff
@@ -63,9 +63,7 @@
               "type": "string"
             }
           },
-          "required": [
-            "en"
-          ]
+          "required": ["en"]
```

Comment:
We need to stop reformatting this file.
Check last two commits in the history of this file. It is going back and forth.
Please introduce some pretty rule with automated check/pre-commit hook or exclude it from this PR